doc/source/conf.py

```diff
@@ -35,7 +35,7 @@
 
 # General information about the project.
 project = u'Bandit'
-copyright = u'2016, Bandit Developers'
+copyright = u'2019, Bandit Developers'
```

Comment:
See comment here: https://github.com/PyCQA/bandit/pull/460#issuecomment-468898501

Reply:
Guess that article differs from what I've heard over the years.  Typically when a file is changed "significantly" you would update the copyright year.  In this case, the docs haven't changed much so I guess I can drop this.